Index: a/app/views/issues/show.html.erb
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- a/app/views/issues/show.html.erb	(revision 5fa8e6df1596527106f2e3918e9eb3bc1b5190e8)
+++ b/app/views/issues/show.html.erb	(date 1590417626521)
@@ -118,6 +118,8 @@
 
 </div>
 
+<%= call_hook(:view_issues_show_after_details, :issue => @issue) %>
+
 <% if @changesets.present? %>
 <div id="issue-changesets">
 <h3><%=l(:label_associated_revisions)%></h3>
